frontend/src/pages/buybacks/BuyBackList.tsx

```diff
@@ -144,50 +117,32 @@ export default function BuyBackList() {
       detailPageURL="/book-buybacks/detail/"
       whitespaceSize={tableWhitespaceSize}
       isNoPagination={isNoPagination}
+      setIsNoPagination={setIsNoPagination}
       isLoading={isLoading}
       setIsLoading={setIsLoading}
       totalNumberOfEntries={numberOfBuyBacks}
       setTotalNumberOfEntries={setNumberOfBuyBacks}
       rows={buybacks}
       APISortFieldMap={APIBBSortFieldMap}
       callGetAPI={callAPI}
-      paginatorLeft={noPaginationSwitch}
+      paginatorLeft={<></>}
```

Comment:
I think we can just make paginatorLeft an optional field so we don't have to pass in nothing